test/e2e/testnet/node.go

```diff
@@ -32,7 +33,7 @@ const (
 	grpcPort       = 9090
 	prometheusPort = 26660
 	tracingPort    = 26661
-	dockerSrcURL   = "ghcr.io/celestiaorg/celestia-app"
+	dockerSrcURL   = "ghcr.io/01builders/celestia-app" // TODO: revert to celestiaorg
```

Comment:
```suggestion
	dockerSrcURL   = "ghcr.io/celestiaorg/celestia-app"
```

Reply:
cc. @chatton

Reply:
also part of #82 these images won't exist in `celestiaorg` until the pr is actually merged.